### ISSUE TITLE
 rabbitmq_delayed_message_exchange: Plugin doesn't support current server version.

### ISSUE THREAD

**Issue by mihaisl** [rabbitmq/rabbitmq-delayed-message-exchange#177]
When I run the command: `rabbitmq-plugins enable rabbitmq_delayed_message_exchange`

I get the following error: 

```
Enabling plugins on node rabbit@localhost:
rabbitmq_delayed_message_exchange
Error:
Failed to enable some plugins: 
    rabbitmq_delayed_message_exchange:
        Plugin doesn't support current server version. Actual broker version: "3.9.6", supported by the plugin: ["3.7.0-3.7.x",
                                                                                                                 "3.8.0-3.8.x"]
```

**Comment by 3101349594** [comment]
me to.  Do you know how to do with it ?

**Comment by mihaisl** [comment]
@3101349594 yes, please follow this thread: https://github.com/rabbitmq/rabbitmq-delayed-message-exchange/issues/104

**Comment by 3101349594** [comment]
> @3101349594 yes, please follow this thread: #104

Thank you ! I hava fixed it by using the plugin of 3.9.0 edition.
